blog.md

```diff
@@ -1,26 +1,78 @@
 ## About
 
-This blog post describes (and is hosted on) a demonstration of a CI/CD pipeline where each Pull Request creates an ephemeral environment that could be reviewed as part of the github review process as well as automatically deploying and upgrading a staging and a production environment.
+Docker and its nascent ecosystem are interesting, but they aren't solving problems -  just moving them around. For me the real problems get solved when we change the discussion from Infrastructure (as a Service) to Platform (as a Service) and the _unit_ delivered by the Platform being a running application rather than a Virtual Machine.
```

Comment:
I've chewed over this opening for a good while now. I'd like to propose an alternative that keeps the message but just wraps it differently. Consider:

« Docker and the ecosystem around it have done some great things for developers, but from an operational standpoint, it's mostly just the same old issues with a fresh coat of paint. Real change happens when we change our perspective from from _Infrastructure_ (as a Service) to _Platform_ (as a Service), and when the ultimate deployment artifact is a _running application_ instead of a virtual machine. »

This retains the punch while also introducing the core thesis of the article. What do you think?